src/routes/users.js

```diff
@@ -3,28 +3,24 @@ const db = require('../db')
 
 const router = express.Router()
 
-const fetchUsers = () => {}
+const fetchUsers = (req, res) => {
+    res.send('Hello')
+}
 const createUsers = () => {}
 const getUser = async (req, res) => {
     const { id } = req.params
     const { rows } = await db.query('SELECT * FROM users WHERE id = $1', [id])
     res.send(rows[0])
 }
+
 const updateUser = () => {}
 const deleteUser = () => {}
-const checkId = (req, res, next , val ) => {
-    console.log(`user id is ${val}`)
-    next()
-}
 
-router.param('id', checkId)
-router.route('/')
-.get(fetchUsers)
-.post(createUsers)
+router.route('/').get(fetchUsers)
+router.route('/').post(createUsers)
 
-router.route('/:id')
-.get(getUser)
-.patch(updateUser)
-.delete(deleteUser)
+router.route('/:id').get(getUser)
```

Comment:
@beblicarl why this change from how it was?

Reply:
you commented on how it was all on one line so I assumed you wanted it to be multilined @jocrah

Reply:
haha okay got it.

i only asked a question to understand why the change was made. I think what was discussed was to reset the changes to even exclude this file because if it should be multi-lined, what was already there before the pr had covered that. This is a bit different and more verbose

Reply:
yea and I understand the assumptions but just a heads up that my questions don't really mean you should perform an action or make a change, it's just a question. If there's an explicit change, I will let you know

Reply:
> haha okay got it.
> 
> i only asked a question to understand why the change was made. I think what was discussed was to reset the changes to even exclude this file because if it should be multi-lined, what was already there before the pr had covered that. This is a bit different and more verbose

okay, I will reset it then

Reply:
> yea and I understand the assumption but just a heads up that my questions don't really mean you should perform an action or make a change, it's just a question. If there's an explicit change, I will let you know

okay

Reply:
> haha okay got it.
> 
> i only asked a question to understand why the change was made. I think what was discussed was to reset the changes to even exclude this file because if it should be multi-lined, what was already there before the pr had covered that. This is a bit different and more verbose

with regards to this, I have reset it and pushed it to the target branches now